### DISCUSSION TITLE
LethalSettings problem

### DISCUSSION THREAD

**Discussion by vasisku** [qwbarch/mirage#118] [Q&A]
sup, i dont have a mirage option in lethalsettings mod, how to fix it?

**Comment by qwbarch** [comment]
Can you provide your log file and a profile code?

**Reply by vasisku** [reply]
where i can find log file and profile code?

**Reply by qwbarch** [reply]
If you're on Thunderstore Mod Manager or r2modman, you can go to ``Settings`` and then look up the following:
- export profile as a code
- copy log file contents to clipboard

**Reply by vasisku** [reply]
meh, i am not using thunderstore manager, i am usually downloading with manual download

**Comment by qwbarch** [comment]
The issue is that you don't have it installed properly then.
I highly recommend you download r2modman so it installd the mod properly.

**Reply by vasisku** [reply]
are there no other options? I just have a lot of mods already downloaded, and I don't want to switch to r2modman to download them again, especially since I haven't downloaded it before

**Reply by qwbarch** [reply]
The only thing I can recommend is to double check for any missing dependencies (some packages depend on other packages). Other than that, it could be an incorrect file structure.

Using a mod manager is always more recommended though. Installing the mod only becomes a 1-button click, and others are able to test your profile out by using your profile code.
